components/layout_2020/flow/inline.rs

```diff
@@ -707,6 +734,27 @@ fn layout_atomic(
         },
     };
 
+    if fragment.content_rect.size.inline + pbm_sums.inline_sum() >
+        ifc.containing_block.inline_size - ifc.inline_position &&
+        ifc.current_nesting_level.white_space.allow_wrap() &&
+        ifc.current_nesting_level.fragments_so_far.len() != 0
+    {
+        ifc.refresh_newline(layout_context);
+    }
+
+    ifc.inline_position += pbm_sums.inline_start;
+    let mut start_corner = Vec2 {
+        block: pbm_sums.block_start,
+        inline: ifc.inline_position - ifc.current_nesting_level.inline_start,
+    };
+    if style.clone_position().is_relative() {
+        start_corner += &relative_adjustement(&style, ifc.containing_block)
+    }
+    child_positioning_context
+        .adjust_static_position_of_hoisted_fragments_with_offset(&start_corner);
+    ifc.positioning_context.append(child_positioning_context);
```

Comment:
This would need to become something like:
```suggestion
    if let Some(child_positioning_context) = child_positioning.take() {
        child_positioning_context
            .adjust_static_position_of_hoisted_fragments_with_offset(&start_corner);
        ifc.positioning_context.append(child_positioning_context);
    }
```

Reply:
Will do.